atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -243,26 +243,38 @@ public void compactInternally() {
                 + " Since this can't be automated in your configuration,"
                 + " good practice would be do to occasional offline manual maintenance of rebuilding"
                 + " IOT tables to compensate for bloat. You can contact Palantir Support if you'd"
-                + " like more information.";
+                + " like more information. Underlying error was: {}";
 
         if (config.enableOracleEnterpriseFeatures()) {
             try {
                 conns.get().executeUnregisteredQuery(
                         "ALTER TABLE " + oracleTableNameGetter.getInternalShortTableName(conns, tableRef)
                                 + " MOVE ONLINE");
             } catch (PalantirSqlException e) {
-                log.error(compactionFailureTemplate + " Underlying error was: {}",
+                log.error(compactionFailureTemplate,
                         tableRef,
                         "(Enterprise Edition that requires this user to be able to perform DDL operations)",
                         e.getMessage());
             } catch (TableMappingNotFoundException e) {
                 throw new RuntimeException(e);
             }
-        } else {
-            log.warn(compactionFailureTemplate,
-                    tableRef,
-                    "(If you are running against Enterprise Edition,"
-                    + " you can set enableOracleEnterpriseFeatures to true in the configuration.)");
+        } else if (config.enableShrinkOnOracleStandardEdition()) {
+            try {
+                conns.get().executeUnregisteredQuery(
```

Comment:
For Sigfred purposes, could we please get some logging before and after these two queries. That will give us an idea of how long they are taking, and also that they are running.

Reply:
Sure.